components/canvas/Section/Section.css

```diff
@@ -0,0 +1,27 @@
+diamond-section {
+  background: var(--diamond-theme-background);
+  color: var(--diamond-theme-color);
+  height: auto;
```

Comment:
```suggestion
  display: block;
```

Custom elements are inline by default so they need setting to block, but height is auto by default so it can be replaced

Reply:
display block also means width: 100%; can be removed